### PR TITLE
FAQ: can I export receive addresses

### PR DESCRIPTION
**Pull request by MarnixCroes**
https://github.com/zkSNACKs/WalletWasabi/issues/8887

once we have the [_Advanced Usage_ ](https://github.com/zkSNACKs/WasabiDoc/pull/1647)section, the FAQ might be better fit there

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -520,6 +520,11 @@ So when you have Wasabi running, you will be notified about an incoming receivin
 But when Wasabi is offline, it does not listen to the network and it will not know about your unconfirmed transaction when you next launch Wasabi.
 In this case you have to wait until your transaction is confirmed in a block, and based on the [BIP 158 block filters](https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki), Wasabi will download that whole block including your transaction from a random P2P node.
 
+### Can I export all my receive addresses?
+
+Wasabi doesn't provide a way to export all generated addresses, however all addresses (used and unused) are listed by using the [listkeys RPC](/using-wasabi/RPC.md#listkeys).
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:525]:
```suggestion
Wasabi doesn't provide a way to export all generated addresses, however all addresses (used and unused) are listed by using the [listkeys RPC](/using-wasabi/RPC.md#listkeys) command.
```

Comment (yahiheb) [reply docs/FAQ/FAQ-UseWasabi.md:525]: Same as https://github.com/zkSNACKs/WasabiDoc/pull/1655#discussion_r1473599131

Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md:525]: well,
I don't see the improvement here
"listkeys RPC" itself is not a command

Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md:525]: one just needs to do the _listkeys_ Remote Procedure Call.

Comment (yahiheb) [reply docs/FAQ/FAQ-UseWasabi.md:525]: > well,
> I don't see the improvement here

For more clarify.

>"listkeys RPC" itself is not a command

So what is it?

It is not uncommon to see `RPC command`, check these:
https://support.spinetix.com/wiki/RPC_API
https://docs.nano.org/commands/rpc-protocol/
https://help.sap.com/docs/SAP_ASE_SDK/088ddce94cc548f59c20518f64c84e1c/bfebe9e96db61014bf8f040071e613d7.html?version=16.0.3.9

image

**Review by yahiheb — APPROVED** [review]

**Pull request merged**